fastchat/serve/inference.py

```diff
@@ -5,14 +5,23 @@
 
 import torch
 try:
-    from transformers import AutoTokenizer, AutoModelForCausalLM, LlamaTokenizer, LlamaForCausalLM, AutoModel, LlamaForCausalLM
+    from transformers import AutoTokenizer, AutoModelForCausalLM, LlamaTokenizer, LlamaForCausalLM, AutoModel, LlamaForCausalLM, AutoConfig 
 except ImportError:
     from transformers import AutoTokenizer, AutoModelForCausalLM, LLaMATokenizer, LLamaForCausalLM, AutoModel
 
 from fastchat.conversation import conv_templates, get_default_conv_template, SeparatorStyle
-from fastchat.serve.compression import compress_module
+from fastchat.serve.compression import compress_module, compress, CompressionConfig, get_compressed_list, apply_compressed_weight
 from fastchat.serve.monkey_patch_non_inplace import replace_llama_attn_with_non_inplace_operations
 from fastchat.serve.serve_chatglm import chatglm_generate_stream
+import glob
```

Comment:
Please fix the code style for import by following this template
```
# block 1: import built-in modules
import abs
import gc

# block 2: import third-party modules
import torch

# block 3: import fastchat modules
from fastchat.serve.compression import compress_module
```

In each block, sort the packages alphabetically

Reply:
code style fixed

Reply:
@andy-yang-1 You used the wrong order again in `compression.py`